### PR TITLE
 #1052 As a Frontend dev, we should stabilize flaky end-to-end tests

### PR DESCRIPTION
**Pull request by wu87988622**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #1052 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/api/workerApi.js [hunk 1]
```diff
@@ -73,3 +77,9 @@ export const createWorker = async params => {
     handleError(err);
   }
 };
+
+function randomPort() {
+  var min = 5000;
+  var max = 65535;
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+}
```
Comment (konekoya) [inline ohara-manager/client/src/api/workerApi.js]: This should probably move to `utils/` as this function has used in many files :)

File: ohara-manager/client/src/api/workerApi.js [hunk 2]
```diff
@@ -49,10 +49,14 @@ export const fetchWorkers = async () => {
...
 
 export const createWorker = async params => {
+  const brokerRes = await axiosInstance.get(`/api/brokers`);
+  const brokerName = get(brokerRes, 'data.result')[0].name;
   try {
     const url = `/api/workers`;
     const data = {
       name: params.name,
+      jmxPort: randomPort(),
+      brokerClusterName: String(brokerName),
```
Comment (konekoya) [inline ohara-manager/client/src/api/workerApi.js]: Is this necessary to wrap in a `String` constructor?

**Comment by wu87988622** [comment]
retry manager

**Comment by wu87988622** [comment]
retry manager

**Comment by wu87988622** [comment]
retry manager

**Pull request merged**